.storybook/preview.js

```diff
@@ -0,0 +1,10 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
woah so weird. is this basically looking for anything matching `on<X>`?

Reply:
idk this is an storybook autogenerated file 🤔 I think this enables some magic for the new storybook 6 "controls" feature